replay_parser.py

```diff
@@ -49,7 +50,7 @@ def read_replays(self):
         for directory in self.directorys:
             files = glob.glob(directory + os.path.sep + '*wotreplay')
             for i, replay in enumerate(files):
-                if replay in ['replay_last_battle.wotreplay', 'temp.wotreplay']:
+                if replay.endswith('replay_last_battle.wotreplay') or replay.endswith('temp.wotreplay'):

```

Comment:
Instead of the two `endswith` how do you feel about:
```If replay.rsplit (os.path.sep, 1)[-1] in ('replay_last_battle.wotreplay', 'temp.wotreplay'):```

Reply:
seems to work, so no problem with that. just committed